packages/terra-clinical-result/src/terra-dev-site/doc/ClinicalResult/AccessibilityGuide.6.doc.mdx

```diff
@@ -0,0 +1,203 @@
+
+The following sections go into more detail on accessibility considerations for each variant.
+
+----
+
+### Code Considerations
+
+#### Clinical Result Blood Pressure with Additional Displays
+
+Clinical Result Blood Pressure creates a standardized view for single result values added to a patient's medical chart. Along with the blood pressure results, we have the option to display additional details like concept names (Systolic, Diastolic) and the date and time of the results. Visually we can see that this information is related but it is possible that screen reader users may not understand the whole cell as a group.
+
+<Notice variant="important" ariaLevel="5">
+
+It is highly recommended to set the `isBloodPressureGrouped` [prop](https://engineering.cerner.com/terra-ui/components/terra-clinical-result/clinical-result/clinical-result-blood-pressure#clinical-result-blood-pressure-props) to true when additional information is displayed with the blood pressure results. This will semantically relate the blood pressure results and additional displays as a group context.
+
+**Code Example:**
+
+<div aria-label="Example demo">
+  <ClinicalResultBloodPressureExtraDisplays />
+</div>
+<div aria-label="Example code">
+
+    import React from 'react';
+    import { ClinicalResultBloodPressure } from 'terra-clinical-result/lib';
+
+    const bloodpressureResultValue = {
+      id: '111',
+      systolic: {
+        eventId: '111.1',
+        result: {
+          value: '120',
+          unit: 'mmHg',
+        },
+        conceptDisplay: 'Systolic',
+        datetimeDisplay: 'Nov 23, 2019 13:31:31',
+      },
+      diastolic: {
+        eventId: '111.2',
+        result: {
+          value: '80',
+          unit: 'mmHg',
+        },
+        conceptDisplay: 'Diastolic',
+        datetimeDisplay: 'Nov 23, 2019 13:31:31',
+      },
+      isBloodPressureGrouped: true,
+    };
+
+    export default () => <ClinicalResultBloodPressure {...bloodpressureResultValue} />;
+
+</div>
+
+This example shows how to set up the optional `isBloodPressureGrouped` prop in order to group the Blood Pressure results with additional displays.
+
+</Notice>
+
+#### Result Name Header Cell Accessible Names
+
+Result Name Header Cell creates a standardized view for the Result Name and Unit of Measure pair within a flowsheet. In many cases, the displayed result name or displayed unit of measure is an abbreviation. Reading out these abbreviations is not very helpful for screen reader users.
+
+<Notice variant="important" ariaLevel="5">
+
+In order to provide better accessible names for the result name and unit of measure, the `fullResultName` and `fullUnit` [props](https://engineering.cerner.com/terra-ui/components/terra-clinical-result/clinical-result/result-name-header-cell#result-name-header-cell-props) can be used to set the full name of the result name and unit when consuming the Result Name Header Cell component.
```

Comment:
Nit: I notice `optional` is called out in L131, I think it would be nice to mention `optional` here as well for the fullResultName and fullUnit.

Reply:
Updated here: 7c7171b18db8c9568f3f9e07940fcee772432841